README.md

```diff
@@ -65,6 +65,16 @@ to start using Ingress in your Kubernetes cluster.
 Release builds of Kong Ingress Controller can be found on Docker Hub in
 [kong/kubernetes-ingress-controller repository][dockerhub-kic].
 
+At the moment we're providing images for:
+
+- Linux `amd64`
+- Linux `arm64`*
+
+<sub>
+* <code>arm64</code> images will be available starting with next release but
+nightly builds are already available
+</sub>
+
```

Comment:
I think it's OK to leave this out. We may just end up forgetting it but in either case it's OK for the README to be an accurate reflection for `main` but not for the previous releases.

```suggestion
```